mod-source-record-manager-server/src/main/java/org/folio/services/util/EventHandlingUtil.java

```diff
@@ -19,8 +19,11 @@
 import java.io.IOException;
 import java.util.List;
 import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
 
 public final class EventHandlingUtil {
+  private final static ConcurrentMap<String, KafkaProducer<String, String>> KAFKA_PRODUCERS = new ConcurrentHashMap<>();
```

Comment:
1. I don't think that using `ConcurrentMap` is a good idea because it will block Vert.X EventBus
2. Do you really want to change the logic of producers for all logic in a module including Data Import features? Maybe will be better to have this change only for quickMarc.